### ISSUE TITLE
Document that config files moved from /app/config/ (v2.x) to /etc/scanservjs(v3.x)

### ISSUE THREAD

**Issue by stephenpapierski** [sbs20/scanservjs#713]
I'm not sure the best place to note this is, but it caught me off guard that all my configs disappeared when I pulled an update (I learned I moved from v2.x to v3.x). The new documentation notes the updated location, but I didn't see anything in the release notes telling me that the location changed. It didn't take a ton of digging, but it might be a nice note in either the release notes of `v3.02` or as a warning at the top of the `10-configuration.md` documentation.

Also, looks like the `50-development.md` needs an update in the `Mount map configuration files` section to remove the old reference to `/app/config/`.

If you determine you'd like it in the docs files versus the release notes, I'm more than willing to submit a pull request, just let me know. Thanks for all the hard work on v3!

**Comment by ceskyDJ** [comment]
Hi, it is already noted here: https://sbs20.github.io/scanservjs/docs/01-install.html.

**Comment by stephenpapierski** [comment]
That's helpful information. I might still argue that `configuration` or the release notes should point to that. I'll admit that `install` is not a place I looked when an already existing install started to behave strangely.

**Comment by ceskyDJ** [comment]
Ok, that's fair. @sbs20, could you add it to the release notes, too, please?

**Comment by sbs20** [comment]
Fair point. Sorry about this. I thought I'd covered it. I think the moment has passed now though
